### ISSUE TITLE
Training stops with 'NoneType' object has no attribute 'name' 

### ISSUE THREAD

**Issue by dakoller** [trailbehind/DeepOSM#84]
..any idea? or a hint, where I might get deeper information about it?

Kind regards,

Daniel

Training samples: 199
Validation samples: 23
--
Training Step: 899  | total loss: 0.23639
Training Step: 899  | total loss: 0.23639 acc: 0.9115 | val_loss: 0.29569 - val_| Momentum | epoch: 001 | loss: 0.23639 - acc: 0.9115 | val_loss: 0.29569 - val_acc: 0.8261 -- iter: 199/199
--
Training Step: 903  | total loss: 0.36168
Training Step: 903  | total loss: 0.36168 acc: 0.8505 | val_loss: 0.23084 - val_| Momentum | epoch: 002 | loss: 0.36168 - acc: 0.8505 | val_loss: 0.23084 - val_acc: 0.9565 -- iter: 199/199
--
Training Step: 907  | total loss: 0.34085
Training Step: 907  | total loss: 0.34085 acc: 0.8723 | val_loss: 0.11795 - val_| Momentum | epoch: 003 | loss: 0.34085 - acc: 0.8723 | val_loss: 0.11795 - val_acc: 0.9565 -- iter: 199/199
--
Training Step: 911  | total loss: 0.61323
Training Step: 911  | total loss: 0.61323 acc: 0.7900 | val_loss: 0.16038 - val_| Momentum | epoch: 004 | loss: 0.61323 - acc: 0.7900 | val_loss: 0.16038 - val_acc: 0.9565 -- iter: 199/199
--
Training Step: 915  | total loss: 0.55020
Training Step: 915  | total loss: 0.55020 acc: 0.7925 | val_loss: 0.23164 - val_| Momentum | epoch: 005 | loss: 0.55020 - acc: 0.7925 | val_loss: 0.23164 - val_acc: 0.9565 -- iter: 199/199
--
_WARNING:tensorflow:Error encountered when serializing data_augmentation.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'NoneType' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing summary_tags.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'dict' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing data_preprocessing.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'NoneType' object has no attribute 'name'_

**Comment by spgriffin** [comment]
I'm experiencing the same issue.  Any suggestions or hints welcome.  Thanks. -Sean

Training samples: 226
Validation samples: 26
--
Training Step: 909  | total loss: 0.20365
| Momentum | epoch: 001 | loss: 0.20365 - acc: 0.9347 | val_loss: 0.09417 - val_acc: 0.9615 -- iter: 226/226
--
Training Step: 913  | total loss: 0.21830
| Momentum | epoch: 002 | loss: 0.21830 - acc: 0.9300 | val_loss: 0.13280 - val_acc: 0.9615 -- iter: 226/226
--
Training Step: 917  | total loss: 0.23635
| Momentum | epoch: 003 | loss: 0.23635 - acc: 0.9105 | val_loss: 0.12314 - val_acc: 0.9231 -- iter: 226/226
--
Training Step: 921  | total loss: 0.23370
| Momentum | epoch: 004 | loss: 0.23370 - acc: 0.9111 | val_loss: 0.14841 - val_acc: 0.9231 -- iter: 226/226
--
Training Step: 925  | total loss: 0.23422
| Momentum | epoch: 005 | loss: 0.23422 - acc: 0.9078 | val_loss: 0.12482 - val_acc: 0.9615 -- iter: 226/226
--
WARNING:tensorflow:Error encountered when serializing data_augmentation.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'NoneType' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing summary_tags.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'dict' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing data_preprocessing.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'NoneType' object has no attribute 'name'

**Comment by spgriffin** [comment]
This issue was discussed here initially, #75.  It was suggested that these are just error messages and that the real problem is here, #79.
